src/parser/tests/expression/TemplateStringExpression.spec.ts

```diff
@@ -74,6 +74,13 @@ describe('TemplateStringExpression', () => {
             program.dispose();
         });
 
+        it('properly transpiles simple template string with no leading text', () => {
+            testTranspile(
+                `a = \`\${one},\${two}\``,
```

Comment:
You can use single quotes for this string so that you don't need to escape the template string stuff. 

```
'a = `${one},${two}`',

Reply:
I fixed this.